model_compression_toolkit/core/keras/keras_implementation.py

```diff
@@ -448,9 +449,10 @@ def get_node_distance_fn(self, layer_class: type,
                 return compute_kl_divergence
             elif node_type_name == SIGMOID:
                 return compute_cs
-        elif layer_class == tf.nn.softmax:
+        elif layer_class == tf.nn.softmax or layer_class == keras.layers.Softmax \
+                or (layer_class == TFOpLambda and SOFTMAX in framework_attrs[keras_constants.FUNCTION]):
             return compute_kl_divergence
-        elif layer_class == tf.nn.sigmoid:
+        elif layer_class == tf.nn.sigmoid or layer_class:
```

Comment:
Are you sure this condition does what you meant it to do?

Reply:
No, this was added by accident